### DISCUSSION TITLE
Adding helm to NonGNU ELPA

### DISCUSSION THREAD

**Discussion by phikal** [emacs-helm/helm#2476] [General]
Hi,

I would like to add Helm to [NonGNU ELPA](https://elpa.nongnu.org) ([see also](https://git.savannah.gnu.org/cgit/emacs/elpa.git/tree/README.org?h=elpa-admin)).  From what I see, everything should work without any changes on your end, so I just want to check if you are OK with this, or if you have any questions/concerns?

**Comment by thierryvolpiatto** [comment]
phikal ***@***.***> writes:

> Hi,
>
> I would like to add Helm to NonGNU ELPA (see also). From what I see,
> everything should work without any changes on your end, so I just want
> to check if you are OK with this, or if you have any
> questions/concerns?

I am OK to add Helm to NonGnu.
Let me know if you need something on my end.

Thanks.

-- 
Thierry

**Comment by phikal** [comment]
Thierry Volpiatto ***@***.***> writes:

> phikal ***@***.***> writes:
>
>> Hi,
>>
>> I would like to add Helm to NonGNU ELPA (see also). From what I see,
>> everything should work without any changes on your end, so I just want
>> to check if you are OK with this, or if you have any
>> questions/concerns?
>
> I am OK to add Helm to NonGnu.

Great, thank you for your quick response.

> Let me know if you need something on my end.

It seems like there are no problem, but if there turns out to be
something, I will get back to you.

> Thanks.
>
> -- 
> Thierry

-- 
	Philip Kaludercic

**Comment by thierryvolpiatto** [comment]
I am discovering Helm has now landed on NonGnu Elpa by reading issue #2480. @phikal can you have a look at this issue? Thanks.

**Comment by phikal** [comment]
Thierry Volpiatto ***@***.***> writes:

> I am discovering Helm has now landed on NonGnu Elpa by reading issue
> #2480. @phikal can you have a look at this issue? Thanks.

I have commented on the issue here: https://debbugs.gnu.org/cgi/bugreport.cgi?bug=53805#8

-- 
	Philip Kaludercic

**Comment by thierryvolpiatto** [comment]
Philip ***@***.***> writes:

> Thierry Volpiatto ***@***.***> writes:
>
>> I am discovering Helm has now landed on NonGnu Elpa by reading issue
>> #2480. @phikal can you have a look at this issue? Thanks.
>
> I have commented on the issue here: https://debbugs.gnu.org/cgi/bugreport.cgi?bug=53805#8

Ok, I see thanks.
So it is related to the issue I ask to be fixed since years, that is
using a pkg.el file if present and fetching infos from it instead of
creating another pkg.el file based on source files headers.
It seems it is what your patch is doing.

-- 
Thierry
